data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -302,11 +302,50 @@ t_rp_meca_path_update AS (
   WHERE rn=1
 ),
 
+t_manual_preprint_match_for_published_date AS (
+  SELECT
+    * EXCEPT(rn),
+  FROM
+  (
+    SELECT 
+      *,
+      ROW_NUMBER() OVER(PARTITION BY long_manuscript_identifier ORDER BY imported_timestamp DESC) AS rn
+    FROM `elife-data-pipeline.prod.unmatched_manuscripts`
+    WHERE preprint_published_at_date IS NOT NULL 
+      AND preprint_published_at_date != ''
+  )
+  WHERE rn = 1
+),
+
+t_europepmc_preprint_publication_date AS (
+  SELECT
+    response.doi,
+    response.firstPublicationDate,
+    CASE
+      WHEN url_list.url LIKE '%10.1101/%v%'
+        THEN REGEXP_EXTRACT(url_list.url, r'10\.\d{3,}.*v([1-9])')
+      WHEN url_list.url LIKE '%researchsquare.com/article/rs-%v%' OR url_list.url LIKE '%arxiv.org/abs/%v%'
```

Comment:
Doesn't research square contain the version in the DOI?
And we are excluding the other option, arxiv.